client/src/App.js

```diff
@@ -1,12 +1,27 @@
 import React from "react";
 import GlobalStyle from "./GlobalStyles";
 import { Launch } from "./pages/Launch";
+import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
+import { MainMenu } from "./pages/MainMenu";
+import { Rides } from "./pages/Rides";
```

Comment:
Those should be default exports

Reply:
Changing it in a later Update, since its quite a bit of work. Just changing `export` to `export default` does not seal the deal. `default export` ist not working either.